cli/utils.py

```diff
@@ -195,6 +201,7 @@ def string_to_list(value, separator=',', append_to_list=True):
             value = [value, ]
     return value
```

Comment:
This method converts string to the list. So the 'value' is a string and immutable. I will keep return there.